capa/features/extractors/ida/insn.py

```diff
@@ -113,7 +113,8 @@ def extract_insn_api_features(f, bb, insn):
 
     if target_func.flags & idaapi.FUNC_LIB:
         name = idaapi.get_name(target_func.start_ea)
-        yield API(name), insn.ea
+        # IDA uses _ prefix for library functions identified using FLIRT
+        yield API(name.lstrip("_")), insn.ea
```

Comment:
how about yielding stripped and unstripped string?

Reply:
could ~ double the feature set though, however, we have no easy way to test performance right now

Reply:
```suggestion
        if name.startswith("_"):
            yield API(name.lstrip("_")), insn.ea
```

Reply:
sure, there are more features, though at most 2x API features, and of those APIs, only those that start with `_`. so probably only a small increase in feature count.

personally, i think its worthwhile to increase expressivity in this way.